src/StockportGovUK.NetStandard.Gateways.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <Title>A package that provides base gateways using HttpClient</Title>
-    <VersionPrefix>7.5.1</VersionPrefix>
+    <VersionPrefix>7.7.0</VersionPrefix>
```

Comment:
The changes below are breaking as they change the type required Inna gateway call, because of this we need to bump the major version.